go/vt/workflow/sleep_workflow.go

```diff
@@ -18,11 +18,12 @@ package workflow
 
 import (
 	"encoding/json"
-	"flag"
 	"fmt"
 	"sync"
 	"time"
 
+	"github.com/spf13/pflag"
+
 	"context"
```

Comment:
can you move this up to the top? (this is an artifact that this used to be an `x/tools` package (so not with the stdlib imports) and someone did a mass-`sed` to replace them))

Reply:
> LGTM, two comments:
> 
> 1. did you see my comment/question on [#10991 (comment)](https://github.com/vitessio/vitess/issues/10991#issuecomment-1211803637) ?
> 2. if we can't delete it, i don't care (much, but i _do_ still care a little) about the rest of the imports, but i do care about `context` heart

Moved the context import. The deprecated flags can be removed when the refactor actually happens.